# Description of Key Laboratory Spaces and Research Equipment {#sec-grant-equipment}

![Willamette River - photo by Mark Currey](images/willamette_header.jpg)

::: {.callout-note}
**Purpose:** These descriptions can be utilized for grant proposals

**Last updated:** 10 May 2021
:::

## CRESKO RESEARCH LABORATORY {#sec-cresko-lab}

### Laboratory {#sec-cresko-lab-space}

The Cresko research group occupies one main lab (1050 sq. ft.), an adjacent stickleback aquatic facility (690 sq. ft.), and a shared fluorescence microscopy room, all on the third floor of Pacific Hall. On the same and adjoining floors, and contiguous with our space, are nine other labs studying evolutionary genetics The Cresko research group occupies one main lab (1050 sq. ft.), a cell culture facility with sterile hood (225 sq. ft.), an adjacent stickleback aquatic facility (690sq. ft.), and a shared fluorescence microscopy room, all on the third floor of Pacific Hall.

### Colleagues {#sec-cresko-colleagues}

::: {.callout-tip}
One of the most important resources is colleagues.
:::

The P.I.'s lab is adjacent to labs of six other evolutionary geneticists: N. Singh (evolutionary genomics in Drosophila), B. Bohannan (bacterial evolution), W. Bradshaw (genetic basis of evolutionary change in *W. Smithii*), M. Barber (protein evolution), K. McGuire (tropical ecology), P. Phillips (evolutionary genetics using Caenorhabditis), P. Ralph (population genomics), M.Streisfeld (evolutionary genetics of Mimulus), and also John Conery (computational biology of gene and genome evolution).

In addition, the Institutes of Molecular Biology (IMB) and Neuroscience (ION) contain numerous researchers with particular interest in the evolution and developmental mechanisms, including K. Guillemin (host-bacterial co-evolution), E. Johnson (genomics of stress response systems), C. Kimmel (developmental genetics), J. Postlethwait (genome evolution), J. Eisen (host-microbe developmental interactions), R. Parthasarathy (live imaging of host-microbe interactions), A. Powell (genomics and tissue modeling in gut development), and T. Ursell (cell mechanics, behavior, modeling).

Monday morning research meetings, Monday IE2 seminars and META center meetings, Tuesday IMB and Thursday ION seminars, Wednesday IE2 tea time, Friday evo-devo biology and META center journal club meetings, and encounters in the hall all bring us and our students in frequent contact with one another.

### Offices {#sec-cresko-offices}

Dr. Cresko's office (200 sq. ft.) is on the same hall as the lab and stickleback facility. Key personnel have dedicated desk space in the laboratory or adjoining offices.

### Computer {#sec-cresko-computer}

The Cresko Lab has several Macintosh, Windows and Linux computers, including a dedicated computer cluster (48 cores, 541GB of memory, 67TB local general purpose storage in a hardware RAID60 configuration, 64TB local backup storage in two hardware RAID6 configurations) for analyzing large data sets. All computers are networked to the university system via 100G connections.

::: {.callout-note}
## Community Accessible Services

The Cresko Lab also hosts several community accessible services:

- **OMERO image server** (<http://omero.uoregon.edu>) serves as a repository of cross-referenced stickleback microsopy images for the Cresko Lab and collaborators
- **Metagenomics online database** (<http://metagenomics.uoregon.edu/metagenomicsdb>), which was developed in house to serve the Microbial Ecology and Theory of Animals (META) Center for Systems Biology (<http://meta.uoregon.edu/>)
- **Stickleback Database** (<http://sticklebackdb.uoregon.edu:8080/sticklebackdb/>) was developed in house to track stickleback crosses and collections from the wild
- **RNA-seqlopedia** (<http://rnaseq.uoregon.edu>) was written by the Cresko Lab and provides an overview of RNA-seq and of the experimental design choices necessary to carry out a successful RNA-seq project
:::

### Animal Facility {#sec-cresko-animal}

The Cresko Lab operates a newly renovated stickleback aquatic facility (690 sq. ft.) capable of housing up to 4000 adult fish and 144 individualized genetic crosses, and contains experiment space specially designed for microscopy and microinjection of embryos for CRISPR, and production of gnotobiotic stickleback (i.e., fish with experimenter-defined microbial communities, including fish with no microbes, or with one or more microbial strains).

::: {.callout-tip}
The facility is managed by Mark Currey (senior research assistant), who designed and maintains the husbandry systems and databases, and directs a staff of animal care technicians.
:::

The facility, located in the Institute of Ecology and Evolution adjacent to the Cresko Lab, is completely enclosed allowing for the care and maintenance of multiple lines of stickleback that have been derived from natural populations. Most of these lines have been in continuous culture since 2003 with current stocks representing over 12 generations in the facility.

Two of the rooms are used to house animals and each room is outfitted with a dedicated recirculating aquaculture system. Lighting in these rooms is manipulated so that one room is set up for a summer lighting regime and the other a winter lighting regime. These lighting regimes allow for the conditioning of the fish to breed (summer regime) and for growing the fish from fry to adults (winter regime). These facilities allow for the ability to perform specific crosses in a controlled environment at precise times so that development can be standardized and quantified.

The third room is used for experiments, and to store and prepare food and husbandry supplies. Half of this room is dedicated for experimental manipulations such as the microinjection procedures of CRISPR or for generating gnotobiotic fish.

::: {.callout-important}
This is one of several model fish facilities at the University of Oregon, and helps maintain compliance for the university's AAALAC accreditation. UO, known as the birthplace of zebrafish research, has a full-time staffed zebrafish facility (see Aquatic Animal Care Services). Expertise in fish husbandry gathered by developing the zebrafish system at Oregon augments and supports the successful husbandry of stickleback.
:::

### Equipment {#sec-cresko-equipment}

Microscopes (fluorescent, compound, dissecting, confocal), centrifuges, sonication and bead beating apparatus, cold rooms, freezers, thermocyclers, spectrophotometers and fluorometers, -80C and liquid nitrogen freezers, and other equipment are either in our lab or available nearby as shared equipment.

::: {.callout-note}
The Cresko Lab has an Arcturus Laser Capture Microdissection (LCM) microscope that has both an Infrared (IR) and ultraviolet (UV) laser that can be used to ablate or to capture microdissected tissues or small numbers of cells, from fixed sections for downstream DNA and RNA analyses.
:::

The Cresko Lab's stickleback facility houses equipment for performing transgenic and gene knockout experiments: a microinjection needle puller and two microinjection systems, including dissection microscopes, micromanipulators, and electronic pressure injection controller.

**The following equipment is available in the Cresko lab:**

- Multiple microcentrifuges and swinging bucket centrifuges
- Sonication and bead beating apparatus
- Cold rooms, refrigerators, and freezers (-20, -80C, and liquid nitrogen)
- Thermocyclers and gel electrophoresis equipment
- Spectrophotometers and fluorometers
- A shared microscopy facility with several compound and dissecting microscopes
- Tissue culture facility equipped with laminar flow hood, incubators, culture media, and sterilizing facility
- Several Macintosh, Windows and Linux computers, including a cluster (48 cores, 541GB of memory, 67TB local general purpose storage in a hardware RAID60 configuration, 64TB local backup storage in two hardware RAID6 configurations)

## THE PREVENTION SCIENCE INSTITUTE RESEARCH LABORATORY {#sec-psi-lab}

PSI is a multidisciplinary research institute in the UO's Millrace Building at Riverfront Research Park. The PIs' and Co-Is' research teams has access to office spaces, treatment/assessment rooms wired with highly advanced audio/video recording equipment linked to a central monitoring room, social neuroscience lab, a waiting room/play area, interactive video conferencing, coding stations, work stations, and conference rooms.

**The following equipment is available in the PSI space:**

- Numerous computer workstations
- 10 servers support data management, analysis, scientific writing, and general office work
- Software licenses for Noldus, SAS, SPSS, and Mplus
- MS Office Suite
- Complete setups for video and teleconferencing
- Document disposal services through a material destruction firm

## COMPUTER SUPPORT AND SECURITY {#sec-computer-security}

UO operates a centralized data and authentication system. All faculty and staff have accounts and have direct access on campus or via VPN from off campus. Internet access is provided at no cost. Data access and transfer capacity are excellent. Computer support is available from the university's Computing Center and from in-house IT staff at PSI. The Computing Center also employs statistical and computer consultants to assist faculty and staff with other computer needs.

### Computers {#sec-computers}

**The Cresko Lab** has several Macintosh, Windows and Linux computers, including a dedicated computer cluster (48 cores, 541GB of memory, 67TB local general purpose storage in a hardware RAID60 configuration, 64TB local backup storage in two hardware RAID6 configurations) for analyzing large data sets. All computers are networked to the university system via 100G connections. The Cresko laboratory also has a laboratory account on Talapas, UO's top 500 computer cluster (see @sec-racs below).

**Servers:** PSI's primary data storage is held on an enterprise-grade 60TB, highly available, RAID 6 array configuration. Data backups are performed each evening. For usage and processing of data, PSI maintains several available workstations with ample software, from word processing to statistical programs. The PSI servers are port-restricted from the rest of the UO network using high-end network firewall appliances and are accessible only through secure means. All data files are protected by a strict access control rules, and access is provided to only a small group of trusted users who require it, thus reducing the exposure of sensitive data. If further data-security is needed, PSI has a special securely-encrypted storage area that has further restrictions in place to prevent any unauthorized data transmissions.

**Zoom:** UO faculty members, staff, and students have access to Zoom videoconferencing services. For UO Zoom users who handle HIPAA-regulated data, a HIPAA-compliant environment is available that includes Zoom, which establishes strict access control rules and strong AES256 encryption to prevent transmission of any PHI (protected health information).

**VPN:** PSI and Cresko lab faculty and staff have access to a VPN connection, when off campus, to servers. The VPN utilizes Cisco AnyConnect platform and is secured with end-to-end strong AES256, and then TLS1.2 for any key exchange or authentication. VPN Users are placed into a specific network pool that is then further port restricted from any critical systems. This gives VPN users the benefit of still being able to access their data, where allowed, but prevents VPN users from accessing any critical administration items or underlying infrastructure.

**Workstations:** PSI has made available workstation and laptops available to its faculty and staff, so as to allow them to continue their work confidently and securely. All PSI-owned workstations are security controlled through a strong set of policies and authentication mechanisms that maintain security for the workstation when interfacing with PSI servers. Drive and transport encryption is employed on the workstations.

## COVID-19-MAP CLIA TESTING LABORATORY {#sec-clia-lab}

This facility is housed in newly renovated laboratory spaces (1400 sq. ft.) and includes a BSL-2 molecular biology lab. The main lab room (1100 sq. ft) is use for all pre-PCR sample handling. An adjacent small lab (300 sq. ft) is dedicated to qPCR and post-PCR processes. A third room set up for freezer storage, with backup power and cooling and with constant monitoring, houses the -80 freezer.

**The following equipment is dedicated to SARS-CoV-2 diagnostic testing:**

- Four Hamilton Vantage 2.0 liquid handling robot with computer stations
- Sixteen QuantStudio5 qPCR thermocycler with two 384-well heads
- Two Proflex thermocyclers each with two 384-well heads
- Hamilton DeCapper SL
- Integra Biosciences 96-channel pipet
- Qubit fluorometer
- Two biosafety cabinets
- Fume hood
- Two Refrigerated Centrifuges (micro and swinging bucket)
- Sterilizing oven
- Refrigerator
- Freezers: -20 and -80 degrees Celsius
- Several sets of single and multichannel micropipettors
- Numerous computer workstations with controlled access to the LIMS and instrumentation

::: {.callout-important}
This facility is housed in newly renovated laboratory spaces, and includes a BSL-2 molecular biology lab equipped with a Hamilton Vantage 2.0 liquid handling robot capable of processing thousands of crude lysate samples from swabs and saliva into qPCR or PCR-ready reactions per day.
:::

The lab is also equipped with two biosafety cabinets, fume hood, centrifuges, sterilizing oven, refrigerator and freezer. The lab is also acquiring a Hamilton DeCapper SL to further streamline handling of sample vials. An adjacent second lab is dedicated to running qPCR and post-PCR processes. This lab is physically and procedurally separate by design to eliminate possible contamination of the pre-PCR BSL2 lab space with PCR amplification products, a known source of false positive results.

This second lab is equipped with a QuantStudio5 qPCR thermocycler with two 384-well heads, two Proflex thermocyclers each with two 384-well heads, and an Integra Biosciences 96-channel pipet. A third room nearby houses a -80C freezer for storage of samples and reagents. This freezer room was built specifically for this purpose, and has redundant power and HVAC as well as continuous monitoring for freezer temperatures to be able to call out to laboratory staff.

::: {.callout-note}
The first lab is designed to be a CLIA-compliant workplace with unidirectional workflow. A Certificate of Conformity (#38D2184568, issued 5/20/2020) has been obtained with the oversight of clinical pathologist Dr. Denis McCarthy, MD, and the Laboratory Director. The activities of the CLIA laboratory are under the direction of Ariana White in her role as General and Technical Supervisor, a role she has had previously in other healthcare settings.
:::

All laboratory processes meet the conditions of CLIA (2003) for molecular pathology, and it includes a custom LIMS built by L7 informatics (<https://l7informatics.com/>) that is fully HIPAA compliant and has user-specific controlled access. Staff are trained in compliance with HIPAA and reporting to state public health entities will be seamless and quick. The COVID-19-MAP CLIA testing laboratory is part of the multi-laboratory network of the Genetics and Cell Characterization Core Facility (GC3F), as further described below in @sec-gc3f.

## The Office of the Vice President for Research and Innovation {#sec-ovpri}

The Office of the Vice President for Research and Innovation (OVPRI) assists with the institution's emphasis on research success, from development to execution of projects through its Research & Innovation Services. Relevant to this proposal, Research Development Services (services for faculty seeking support for research, performance, public service, and scholarly projects), Sponsored Project Services (proposal budget preparation, institutional clearances, and grant and contract administration) and Research Compliance Services (support and protection for university, faculty, staff, student, and participant interest in research compliance matters), which are divisions of OVPRI, work to support and enhance on-campus research.

## University of Oregon Research Environment {#sec-uo-research-environment}

The University of Oregon (UO) is classified as a Carnegie Doctoral/Research University. It has an extensive history of substantial research, federal grant funding, and scientific inquiry. The College of Arts and Sciences comprises 40 departments and programs, with 463 tenure-track faculty. The research activity of the College of Arts and Sciences faculty is the basis for the UO's status as a Carnegie Research I institution and its membership in the Association of American Universities.

The College of Education at the UO ranks 12th among all graduate institutions of education in the nation, ranks 4th among all public graduate institutions of education in the nation, and ranks 2nd overall for faculty productivity in funded research per faculty member, according to U.S. News & World Report's 2016 website rankings.

The UO includes a network of interdisciplinary centers, research institutes, and outreach units that supports federal grant funding and direct community service. The three MPIs are members of research institutes, as are many of the additional PIs and key personnel. In the past five years, four UO faculty have been elected to the National Academy of Sciences, three named Sloan Research Fellows, six elected to the American Academy of Arts and Sciences, five named Guggenheim Fellows, seven elected as American Mathematical Society Fellows, and nine elected to the American Association for the Advancement of Science, with an additional faculty member elected as president of the AAAS.

The UO provides comprehensive instructional, research, and public service programs that advance scientific and humanistic knowledge. Research programs serve the educational, cultural, and economic needs of the region and the nation. Administrative units provide direct oversight and support for graduate programs, grant proposal submission, research compliance, contracts and grant administration, and research initiatives. The UO has collaborative research-based relationships with every school district in the state and in many other states in the United States.

## UNIVERSITY OF OREGON RESEARCH CORE FACILITIES {#sec-uo-core-facilities}

The University of Oregon is well known for integrative research, has broad collaborative expertise and houses many shared facilities. A network of Research Core Facilities (RCFs) are managed by the Office of the Vice President for Research and Innovation (OVPRI). This network provides infrastructure and services in many technological areas. Each RCF is staffed by a director and other personnel and are available to researchers from across all parts of the university, and those from outside the university. The RCFs are run in a research compliant fashion using subsidy from the OVPRI for the Director's salary and the proportional salaries of members of Research Core Business Services who are well trained in finance and compliance. Below is a list of the salient RCFs for this project.

### Genomics and Cell Characterization Core Facility {#sec-gc3f}

This facility is directed by Dr. Doug Turnbull and staffed by top genomics personnel, including several full time laboratory technicians and the General and Technical Supervisor for the CLIA laboratory. In anticipation of the growing need to support SARS-CoV-2 testing, the facility is beginning searches for three additional laboratory technicians.

The main laboratory houses Illumina NovaSeq, NextSeq500 and MiSeq massively parallel sequencers and two systems for use in long molecule sequencing: 10x Genomics's platform for producing short-read Illumina-compatible DNA- or RNA-seq libraries starting from nucleic acids or from single cells and a PacBio Sequel II. These platforms are run by Dr. Turnbull and his staff, who also provide sequencing library preparation services, particularly for the PacBio and 10x Genomics platforms. Output data are initially processed by computer scientist Nicholas Stiffler. The facility charges for reagents, personnel time, and service contract. The facility's sequencing, fragment analysis and library preparation services will continue to be a critical part of our research program. This facility also oversees a dedicated biological imaging space that contains specialty microscopes including light sheet, confocals, and the expertise needed to effectively use them. The CLIA testing laboratory described is administratively part of GC3F.

**Equipment:**

- Illumina HiSeq4000, NextSeq500 and MiSeq massively parallel sequencers
- PacBio Sequel2
- 10x Genomics' platforms
- Two flow cytometers (Sony SH800 Cell Sorter and Applied Biosystems Attune Cytometer)
- Eppendorf EPMotion 5075 TMX liquid handling robot
- 128 thread primary analysis server for processing next generation sequencing data
- Suite of advanced light microscopes including a Zeiss Elyra and Nikon Sora for super-resolution imaging
- Sequencing, fragment analysis, and library preparation services and equipment

### Research Advanced Computing Services (RACS) and Talapas high performance cluster {#sec-racs}

Talapas is the University of Oregon's primary supercomputer. It was built in 2017 and is managed by Director Dr. Nick Maggio and his staff. The Cresko laboratory has an unlimited membership to the cluster via a yearly membership. Talapas is a heterogeneous supercomputing cluster capable of performing over 1.2 quadrillion calculations per second and providing 1.5 petabytes (1,500 trillion bytes) of high performance, parallel data storage.

Talapas makes three distinct classes of compute nodes available to researchers: standard compute nodes built around the Intel "Broadwell" and "Skylake" generations of processor, GPU compute nodes equipped with dual nVidia Tesla K80 accelerators, and large memory servers. This large compute cluster has over 200 standard nodes with at least 128GB of memory each, 24 GPU nodes with 256GB memory each, and 8 large memory nodes with 1TB to 4TB of memory each. Talapas computational and storage resources are networked together via EDR Infiniband and connected to Internet2 via a high-speed 100 GigE interconnect. Talapas is open to all UO researchers and students for scientific research.

**Specifications:**

- Heterogeneous supercomputing cluster capable of performing over 250 trillion calculations ever second and providing 1.5 petabytes (1,500 trillion bytes) of high performance, parallel data storage
- Standard compute nodes built around the Intel "Broadwell" generation of processor, GPU compute nodes equipped with dual nVidia Tesla K80 accelerators, and large memory servers
- 96 standard nodes with 128GB of memory each, 24 GPU nodes with 256GB memory each, and 8 large memory nodes with 4TB of memory each
- The cluster presently has 1.5 Petabytes of storage (soon to be 10 Petabytes)

### Aquatic Animal Care Services {#sec-aquatic-animal-care}

This service, directed by Timothy Mason, provides cutting edge animal husbandry and experimental expertise for working with aquatic animals. The facility maintains housing and veterinary care for zebrafish stocks with a capacity of more than a million fish, husbandry training, and cryopreservation of sperm, and provides experiment rooms for microinjection and other research activities.

::: {.callout-important}
UO is the birthplace of zebrafish (Danio rerio) as a model vertebrate, having been pioneered by Dr. George Streisinger in the 1970s. The amassed expertise in zebrafish at this facility and in the many UO research labs it serves has greatly facilitated the development of other emerging aquatic models at the UO, including threespine stickleback and spotted gar.
:::

### Zebrafish International Resource Center (ZIRC) {#sec-zirc}

ZIRC, co-directed by Zoltan Varga and Monte Westerfield, is across the street from our lab, and it provides the world's central repository for wild-type and mutant strains of zebrafish and for materials and information about zebrafish research. They distribute materials and zebrafish strains to the international research community. ZIRC provides pathology services for potentially diseased fish, including those from the Cresko Lab stickleback facility.

### Zebrafish Information Network (ZFIN) {#sec-zfin}

ZFIN (<http://zfin.org>) is an online digital resource that contains and distributes information about zebrafish research to the world. These resources include information about zebrafish genetics, genomics, and development, among many other things. ZFIN also provides a clearing house for information on researchers to encourage collaboration networks. Having this vast repository of information and tools for fish developmental genetics easily accessible greatly enhances our ability to perform evolutionary developmental analysis.

### Biological Imaging Core {#sec-biological-imaging}

This facility brings together specialty microscopes including light sheet, laser capture microdissection, confocals, and the expertise needed to effectively use them. These resources will be critical in the three-dimensional visualization and documentation of transgene expression in living sticklebacks, as well as observation of living microbial endosymbionts in the stickleback gut. The light sheet microscope has already proven itself for studying interactions of microbes in the gut of living zebrafish larvae here at UO, and we will be able to adapt these methodologies to using stickleback larvae.

### Histology and Genetic Modification Core Facility {#sec-histology-core}

This facility, directed by Dr. Ute Hostick, offers cryo-, paraffin and plastic sectioning, and histological staining. These services will be useful for phenotyping of mutant stickleback strains, when characterization by in situ hybridization on cryosectioned tissue or histological staining of fixed, sectioned tissue is required. This cutting edge facility also provides many procedures for generating and strain storage of genetically modified mice, including pronuclear injection, embryo cryopreservation stem cell tissue culture and genotyping.

### Center for Advanced Materials Characterization in Oregon (CAMCOr) {#sec-camcor}

This facility, directed by Kurt Langworthy, houses multiple TEM and SEM microscopes, and many apparatus for NMR, X-ray crystallography, and other specialized microscopy for life sciences, materials science and chemistry applications. In addition, the facility houses resource for nanofabrication and microfluidics. In addition, a new Zeiss Versa X-ray microscope housed in this facility produces exquisite 3-D high resolution scans of biological and non-biological subjects non-destructively.

### Technical Science Administration {#sec-tech-science-admin}

This facility, directed by John Boosinger, houses professional machinists and electrical engineers who help design and build research instrumentation. For the Cresko research group, this has included such diverse things as fish husbandry and microinjection equipment.

### Lewis Center for Neuroimaging {#sec-lewis-center}

This facility, directed by Fred Sabb, supports research in cognitive science and biological imaging and includes a Siemens Skyra functional MRI and staff who design and fabricate MR coils suited to a variety of projects.

### Molecular Biology Kitchen {#sec-mol-bio-kitchen}

This facility provides us with media, plates, sterilized pipette tips, buffers, and clean glassware for molecular biology and bacteriology needs.

### Lab Supply Stores {#sec-lab-supply-stores}

Laboratory supplies and small equipment are available for purchase in a centralized store maintained by Thermofisher in a science building adjacent to ours.
